Project1/Menu.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project1
+{
+    internal class Menu
+    {
+        public void BasicMenu()
+        {
+            Console.WriteLine("Wybierz funkcję w menu:");
+            Console.WriteLine("1. Kalkulator");
+            Console.WriteLine("2. Liczba pierwsza, Silnia, Liczba doskonała, Potęgi");
+        }
+        public void Calculator()
```

Comment:
Tutaj coś też z `Show` bym dał. Chodzi ogólnie o to aby metoda wskazywała co robi np Console.WriteLine nam mówi że w konsoli napisz linie. Tak np mogło by  być `Menu.Show` - także ładnie mówi co jest za tą metodą i za co ona odpowiada.